docs/faq/validator.md

```diff
@@ -5,47 +5,56 @@ description: Frequently asked questions about validation on the camino and colum
 
 # Validator FAQ
 
-## Who can be a Camino validator?
+### Who can be a Camino validator?
 
-Any travel industry market participant who fulfills the following conditions can be a validator on the Camino Mainnet:
+Any participant in the travel industry who meets the following criteria can become a validator on the Camino Mainnet:
 
-- Have at least 100,000 Camino (CAM) tokens locked in their wallet
-- Is KYC / KYB verified
-- Has been admitted as a validator by the validator consortia via a vote
-  The last condition is omitted for Validators who secure the network from the start. To secure the Columbus Testnet, one must only be a market participant in the travel industry.
+- Holds a minimum of 100,000 Camino (CAM) tokens in their wallet.
+- Has undergone and passed a Know-Your-Customer (KYC) and Know-Your-Business (KYB) verification process.
+- Has been elected as a validator by the validator consortium through a voting process.
 
-## Staking Parameters on Camino
+:::note
 
-When a validator is done validating the Camino Mainnet, it receives back the Camino tokens it staked. It may receive a reward for helping to secure the network. A validator only receives a validation reward if it is sufficiently responsive and correct during the validation period.
+Please note, the last requirement is not applicable for validators who are securing the network from its inception. For securing the Columbus Testnet, it's sufficient to be a market participant in the travel industry.
```

Comment:
How's that? I don't think that we'll have power to do something like add/remove c-members without voting, so any validator can be removed later. Isn't this right? May be I got this line wrong

Reply:
Maybe I need better wording. What I was trying to say there was that the initial c-members (in genesis block) would not go through a  *voting process* and they are exempt from that *last requirement*.

Reply:
Updated this. Please note that this commit is part of the PR #71. I don't know what will happen to these comments when #71 is merged and I rebase this PR to the new dev. :)